packages/server/src/migrations/migrate.ts

```diff
@@ -1257,6 +1275,10 @@ function buildIndexSql(tableName: string, indexName: string, index: IndexDefinit
     result += ')';
   }
 
+  if (index.nulls) {
+    result += ' NULLS ' + index.nulls;
```

Comment:
This is the main blocker on Postgres 13, yeah?

Reply:
Correct: this is the way we would like to construct our indexes/queries, but it was only added in Postgres 15